### PR TITLE
Hello-rails-react app features

### PR DESCRIPTION
**Pull request by katfogy**
**In this project, I did the following tasks**

- [x] Created a new Rails app called 'hello-rails-react'
- [x] Created a new project with Webpack
- [x] Installed all dependencies
- [x] Used PostgreSQL in this project
- [x] Created a table called greetings with 5 messages/greetings, created a controller with a route
- [x] Created a static view that will be the root of the app
- [x] Created the App component with react-router and rendered it in the static view
- [x] Created the Greeting component that will display a greeting.
- [x] Created a store, an action, and a reducer that will connect to the API endpoint to get the random greeting.
- [x] Displayed the random greeting in the Greeting component.
- [x] Updated README file
- [x] Fixed linter errors

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
```
Comment (Shedrack-Sunday) [inline db/seeds.rb]: - [ ] Based on project requirements, you're expected to add `5` messages in the seed file, which would be the messages your view will render after action by the message controller

### Instruction:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @katfogy    👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with Rails model✔️
- Kudus making the requirement asked      ✔️  

# - Required Changes  🌵  

- [ ] Kindly take note of the comments i made in the `seed` file to make sure we meet all requirements  👍  

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,4 @@
...
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+greeting_msg=["Hello React Rails", "Welcome Rails React", "Welcome My code Reviewer", "Hello Foga", "Hello Grace"]
+greeting_msg.each do |msg|
+    Greeting.create(message: msg)
+end
```
Comment (TadesseDev) [inline db/seeds.rb:4]: Good job ✅

**Review by TadesseDev — APPROVED** [review]
### Hi @katfogy 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅


## Optional suggestions

- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**